src/lib/spinel/radio_spinel_impl.hpp

```diff
@@ -1677,7 +1677,7 @@ otError RadioSpinel<InterfaceType, ProcessContextType>::WaitResponse(void)
 {
     uint64_t end = otPlatTimeGet() + kMaxWaitTime * US_PER_MS;
 
-    otLogDebgPlat("Wait response: tid=%u key=%u", mWaitingTid, mWaitingKey);
+    otLogDebgPlat("Wait response: tid=%u key=%lu", mWaitingTid, ToUlong(mWaitingKey());
```

Comment:
Should be ToUlong(mWaitingKey) instead of ToUlong(mWaitingKey**()**)

Reply:
Thanks, fixed.